src/main/java/icbm/classic/content/blocks/launcher/screen/TileLauncherScreen.java

```diff
@@ -211,15 +242,20 @@ public IActionStatus launch(IMissileLauncher launcher, int launcherCount, boolea
 
     public boolean fireAllLaunchers(boolean simulate) {
         refreshStatus = true;
-        // TODO add chain fire delay settings to screen
-        boolean hasFired = false;
-        for(IMissileLauncher launcher : getLaunchersInGroup()) {
-            final IActionStatus status = launch(launcher, getLaunchers().size(), simulate); // TODO output status to users
```

Comment:
Eventual goal is to return the error information to the user. Its only boolean for short term. So we need to move the delay to the launcher itself... which was the plan.

Reply:
so the launchers have the (n) delay settable by user instead of a staggered salvo fire?

Reply:
Both, was already considering a manually setup delay per launcher. Then controllers would have the option to encode additional information such as delay. With the launcher still controlling the countdown itself. This would keep the controllers rather isolated from knowing what the launcher is doing. All the controller would do is invoke launch and pass back the messages to the user. So if a delay was set then the user would see "Firing missile at xyz in 3 seconds" and eventually once we add multi-block screens could watch the countdown.

Reply:
Do i prioritize launcher or controller for timer? or do you want it to be an option in the controller?

Reply:
If you can join the discord we can go over things. As the plan I had in mind was to use a new controller block. I'll have to see if I still have notes on the design.